packages/vuetify/src/components/VBtn/VBtn.sass

```diff
@@ -76,7 +76,7 @@
     pointer-events: none
 
     &.v-btn--disabled
-      color: rgba(var(--v-theme-on-surface), $button-disabled-opacity)
+      color: rgba(var(--v-theme-on-surface), $button-disabled-opacity) !important
```

Comment:
Not good idea. On dark background it will be look like this
![изображение](https://user-images.githubusercontent.com/12619075/170609762-17afb8f6-44ce-4294-8ae2-6325c1bab463.png)

Reply:
Thanks for picking up, I will do a deep dive into `variant` tonight